File: src/main/java/shape/Point.java
package shape;

public class Point {

    private double x;
    private double y;

     public Point(double x, double y) {
        this.x = x;
        this.y = y;
     }

     public double GetX() {
         return x;
     }

     public double GetY() {
         return y;
     }

     public Point Clone() {
         return new Point(x, y);
     }

     public void MovePoint(double addXvalue , double addYvalue) {
         this.x += addXvalue;
         this.y += addYvalue;
     }

}

File: src/main/kotlin/shape/Point.kt
package shape

class Point(private var x: Double, private var y: Double) {

    fun GetX(): Double {
        return x
    }

    fun GetY(): Double {
        return y
    }

    fun Clone(): Point {
        return Point(x, y)
    }

    fun MovePoint(addXvalue: Double, addYvalue: Double) {
        x += addXvalue
        y += addYvalue
    }
}